### PR TITLE
Beta docs

### PR DESCRIPTION
**Pull request by snehalkotwal**

### CODE REVIEW

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroup.md [hunk 1]
```diff
@@ -48,101 +52,106 @@ Get-EntraBetaGroup
...
 ```
-```output
-DisplayName Id                                   MailNickname Description GroupTypes
------------ --                                   ------------ ----------- ----------
-UPDISPLAY   fc446647-e8ff-47f1-a489-cf31694c0d35 Remoteliving Upd1        {Unified}
+
+```Output
+DisplayName                                       Id                                   MailNickname Description GroupTypes
+-----------                                       --                                   ------------ ----------- ----------
+SimpleTestGrpbcd1865e-c8b9-4f90-8943-77607ef15590 eeeeeeee-4444-5555-6666-ffffffffffff NickName                 {}
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroup.md:90]: This `DisplayName` has a guid. Remove it

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroup.md [hunk 2]
```diff
@@ -48,101 +52,106 @@ Get-EntraBetaGroup
...
-Parents of Contoso      05b0552e-39cd-4df4-a8f5-00ade912e83d parentsofcontoso      Parents of Contoso
-Contoso Team            0877c6c6-fc99-4d51-9871-8335be7cfc9d contosoteam           A collaboration area for the Cont...
-HelpDesk admin group    0883fd77-0ee8-45de-a21e-f32af1623acc helpDeskAdminGroup    Group assignable to role
-New Employee Onboarding 0a58c57b-a9ae-49a2-824f-8e9cb86d4512 newemployeeonboarding New Employee Onboarding
+
+```Output
+DisplayName                                     Id                                   MailNickname    Description       GroupTypes
+-----------                                     --                                   ------------    -----------       ----------
+SimpleTestGrp                                   aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb NickName                          {}
+SimpleGroupc13adf68-dffb-4185-a94f-e81bda906aa5 bbbbbbbb-1111-2222-3333-cccccccccccc NickName                          {}
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroup.md:106]: This `DisplayName` has a guid. Remove it

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroup.md [hunk 3]
```diff
@@ -48,101 +52,106 @@ Get-EntraBetaGroup
...
+testGroupInAU10                                 cccccccc-2222-3333-4444-dddddddddddd testGroupInAU10 testGroupInAU10   {DynamicMembership, Unified}
+My new group                                    dddddddd-3333-4444-5555-eeeeeeeeeeee NotSet          New created group {}
+SimpleGroupaeaea4bd-30df-4e13-b318-b416ed4e9357 eeeeeeee-4444-5555-6666-ffffffffffff NickName                          {}
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroup.md:109]: This `DisplayName` has a guid. Remove it

**Review by KenitoInc — DISMISSED** [review]

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaApplicationLogo.md [hunk 4]
```diff
@@ -63,10 +67,11 @@ Accept wildcard characters: False
 ```
 
 ### -FilePath
+
 If provided, the application logo is copied with a random filename to the file path that is specified in this parameter
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaApplicationLogo.md:71]:
```suggestion
If provided, the application logo is saved to the specified file path using a random file name.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaApplicationLogo.md [hunk 5]
```diff
@@ -8,50 +19,43 @@ schema: 2.0.0
...
 ```
 
-This example shows how to retrieve the application logo for an application that is specified through the Object ID parameter
+This example shows how to retrieve the application logo for an application that is specified through the Object ID parameter.
 
 ## Parameters
 
 ### -FileName
+
 If provided, the application logo is copied to the file who's name is provided in this parameter
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaApplicationLogo.md:55]:
```suggestion
If provided, the application logo is saved to the file using the specified file name.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md [hunk 6]
```diff
@@ -8,61 +19,155 @@ schema: 2.0.0
...
-3cb87a8f-0a41-4ca8-8910-e56cc00114a3 48445467-033c-42ca-8e38-8d181db1d49c bastias_WindowsPhone_5/1/2016_12:53 PM
+```Output
+DeletedDateTime Id                                   AccountEnabled ApproximateLastSignInDateTime ComplianceExpirationDateTime DeviceCategory DeviceId                             DeviceMetada
+                                                                                                                                                                                   ta
+--------------- --                                   -------------- ----------------------------- ---------------------------- -------------- --------                             ------------
+                bbbbbbbb-1111-1111-1111-cccccccccccc True                                                                                     dddddddd-9999-0000-1111-eeeeeeeeeeee MetaData
 ```
 
-This command gets the specified device.
+This example demonstrates how to retrieve specific device by providing ID. This command gets the specified device.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md:75]:
```suggestion
This example shows how to retrieve a device using its ID.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md [hunk 7]
```diff
@@ -8,61 +19,155 @@ schema: 2.0.0
...
+```Output
+DeletedDateTime Id                                   AccountEnabled ApproximateLastSignInDateTime ComplianceExpirationDateTime DeviceCategory DeviceId                             DeviceMetada
+                                                                                                                                                                                   ta
+--------------- --                                   -------------- ----------------------------- ---------------------------- -------------- --------                             ------------
+                aaaaaaaa-1111-1111-1111-bbbbbbbbbbbb True                                                                                     aaaaaaaa-6666-7777-8888-bbbbbbbbbbbb MetaData
+                bbbbbbbb-1111-1111-1111-cccccccccccc True                                                                                     aaaaaaaa-6666-7777-8888-bbbbbbbbbbbb MetaData
 ```
 
+This example demonstrates how to retrieve all devices from Microsoft Entra ID.  
 This command gets all available devices.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md:93]:
```suggestion
This example demonstrates how to retrieve all devices from Microsoft Entra ID.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md [hunk 8]
```diff
@@ -8,61 +19,155 @@ schema: 2.0.0
...
+```Output
+DeletedDateTime Id                                   AccountEnabled ApproximateLastSignInDateTime ComplianceExpirationDateTime DeviceCategory DeviceId                             DeviceMetada
+                                                                                                                                                                                   ta
+--------------- --                                   -------------- ----------------------------- ---------------------------- -------------- --------                             ------------
+                aaaaaaaa-1111-1111-1111-bbbbbbbbbbbb True                                                                                     aaaaaaaa-6666-7777-8888-bbbbbbbbbbbb MetaData
+                bbbbbbbb-1111-1111-1111-cccccccccccc True                                                                                     aaaaaaaa-6666-7777-8888-bbbbbbbbbbbb MetaData
+```
+
+This example demonstrates how to retrieve top two devices from Microsoft Entra ID.  
+This command gets the two devices from Microsoft Entra ID.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md:111]:
```suggestion
This example demonstrates how to retrieve two devices from Microsoft Entra ID.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md [hunk 9]
```diff
@@ -8,61 +19,155 @@ schema: 2.0.0
...
+
+### Example 4: Get a device by display name
+
+```powershell
+Connect-Entra -Scopes 'Device.Read.All'
+Get-EntraBetaDevice -Filter "DisplayName eq 'AkshayLodha'"
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md:117]:
```suggestion
Get-EntraBetaDevice -Filter "DisplayName eq 'Woodgrove Desktop'"
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md [hunk 10]
```diff
@@ -8,61 +19,155 @@ schema: 2.0.0
...
+
+```Output
+DeletedDateTime Id                                   AccountEnabled ApproximateLastSignInDateTime ComplianceExpirationDateTime DeviceCategory DeviceId                             DeviceMetada
+                                                                                                                                                                                   ta
+--------------- --                                   -------------- ----------------------------- ---------------------------- -------------- --------                             ------------
+                bbbbbbbb-1111-1111-1111-cccccccccccc True                                                                                     dddddddd-9999-0000-1111-eeeeeeeeeeee MetaData
+```
+
+This example demonstrates how to retrieve device by display name from Microsoft Entra ID.  
+This command gets the specified device.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md:128]:
```suggestion
This example demonstrates how to retrieve device using the display name.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md [hunk 11]
```diff
@@ -8,61 +19,155 @@ schema: 2.0.0
...
+
+### Example 5: Get a device filter by display name
+
+```powershell
+Connect-Entra -Scopes 'Device.Read.All'
+Get-EntraBetaDevice -Filter "startswith(DisplayName,'Aksh')"
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md:134]:
```suggestion
Get-EntraBetaDevice -Filter "startswith(DisplayName,'Desktop')"
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md [hunk 12]
```diff
@@ -8,61 +19,155 @@ schema: 2.0.0
...
+```
+
+```Output
+DeletedDateTime Id                                   AccountEnabled ApproximateLastSignInDateTime ComplianceExpirationDateTime DeviceCategory DeviceId                             DeviceMetada
+                                                                                                                                                                                   ta
+--------------- --                                   -------------- ----------------------------- ---------------------------- -------------- --------                             ------------
+                bbbbbbbb-1111-1111-1111-cccccccccccc True                                                                                     dddddddd-9999-0000-1111-eeeeeeeeeeee MetaData
+```
+
+This example demonstrates how to retrieve all the devices whose display name starts with Aksh from Microsoft Entra ID.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md:144]:
```suggestion
This example demonstrates how to retrieve all the devices whose display name starts with the word `Desktop`.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md [hunk 13]
```diff
@@ -8,61 +19,155 @@ schema: 2.0.0
...
+                bbbbbbbb-1111-1111-1111-cccccccccccc True                                                                                     dddddddd-9999-0000-1111-eeeeeeeeeeee MetaData
+```
+
+This example demonstrates how to retrieve all the devices whose display name starts with Aksh from Microsoft Entra ID.  
+
+### Example 6: Search among retrieved devices
+
+```powershell
+Connect-Entra -Scopes 'Device.Read.All'
+Get-EntraBetaDevice -SearchString 'Ashwini'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md:150]:
```suggestion
Get-EntraBetaDevice -SearchString 'DESKTOP'
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md [hunk 14]
```diff
@@ -8,61 +19,155 @@ schema: 2.0.0
...
+
+```Output
+DeletedDateTime Id                                   AccountEnabled ApproximateLastSignInDateTime ComplianceExpirationDateTime DeviceCategory DeviceId                             DeviceMetada
+                                                                                                                                                                                   ta
+--------------- --                                   -------------- ----------------------------- ---------------------------- -------------- --------                             ------------
+                bbbbbbbb-1111-1111-1111-cccccccccccc True                                                                                     dddddddd-9999-0000-1111-eeeeeeeeeeee MetaData
+```
+
+This example demonstrates how to retrieve devices by search string from Microsoft Entra ID.  
+This command gets all devices that match the value of SearchString against the first characters in DisplayName.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDevice.md:161]:
```suggestion
This example demonstrates how to retrieve devices by SearchString.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroup.md [hunk 15]
```diff
@@ -48,101 +51,106 @@ Get-EntraBetaGroup
...
+-----------                                     --                                   ------------    -----------       ----------
+SimpleTestGrp                                   aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb NickName                          {}
+SimpleGroup                                     bbbbbbbb-1111-2222-3333-cccccccccccc NickName                          {}
+testGroupInAU10                                 cccccccc-2222-3333-4444-dddddddddddd testGroupInAU10 testGroupInAU10   {DynamicMembership, Unified}
+My new group                                    dddddddd-3333-4444-5555-eeeeeeeeeeee NotSet          New created group {}
+SimpleGroup                                     eeeeeeee-4444-5555-6666-ffffffffffff NickName                          {}
 ```
 
 This example demonstrates how to get top five groups from Microsoft Entra ID.  
 This command gets the five groups in Microsoft Entra ID.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroup.md:112]:
```suggestion
This example demonstrates how to get top five groups.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroup.md [hunk 16]
```diff
@@ -48,101 +51,106 @@ Get-EntraBetaGroup
...
+```Output
 DisplayName        Id                                   MailNickname     Description        GroupTypes
 -----------        --                                   ------------     -----------        ----------
-Parents of Contoso 05b0552e-39cd-4df4-a8f5-00ade912e83d parentsofcontoso Parents of Contoso {Unified}
+Parents of Contoso aaaaaaaa-6666-7777-8888-bbbbbbbbbbbb parentsofcontoso Parents of Contoso {Unified}
 ```  
-In this example, we retrieve group by display name from Microsoft Entra ID.    
+
+In this example, we retrieve group by display name from Microsoft Entra ID.
 This command gets the specified group.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroup.md:128]:
```suggestion
In this example, we retrieve group using the Display Name.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupAppRoleAssignment.md [hunk 17]
```diff
@@ -55,11 +58,13 @@ The command stores the ID in the $GroupId variable.
 The second command gets the application role assignments of the group in $GroupId.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupAppRoleAssignment.md:58]:
```suggestion
This example retrieves the application role assignments of the group.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupAppRoleAssignment.md [hunk 18]
```diff
@@ -31,17 +31,20 @@ Get-EntraBetaGroupAppRoleAssignment
...
+Connect-Entra -Scopes 'Directory.Read.All'
 $GroupId = (Get-EntraBetaGroup -Top 1).ObjectId
 Get-EntraBetaGroupAppRoleAssignment -ObjectId $GroupId
 ```
 
-```output
+```Output
 ObjectId                                    ResourceDisplayName                 PrincipalDisplayName
 --------                                    -------------------                 --------------------
 MSVrBV4APk--eAGnHqMKBLflsQG3rU1EmDFKvgra41I Microsoft Device Management Checkin Ask HR
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupAppRoleAssignment.md:50]: Kindly remove this part:

The first command gets the object ID of a group by using the [Get-EntraBetaGroup](./Get-EntraBetaGroup.md) cmdlet.
The command stores the ID in the $GroupId variable.

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupLifecyclePolicy.md [hunk 19]
```diff
@@ -8,41 +18,73 @@ schema: 2.0.0
...
+
+```Output
+Id                                   AlternateNotificationEmails GroupLifetimeInDays ManagedGroupTypes
+--                                   --------------------------- ------------------- -----------------
+eeeeeeee-4444-5555-6666-ffffffffffff example@contoso.un          99                  Selected
 ```
 
-This command retrieves the group expiration settings configured for the tenant
+This example demonstrates how to retrieve the properties and relationships of all groupLifecyclePolicies in Microsoft Entra ID.
+This command retrieves the group expiration settings configured for the tenant.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupLifecyclePolicy.md:63]:
```suggestion
This example demonstrates how to retrieve the properties and relationships of all groupLifecyclePolicies in Microsoft Entra ID.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupMember.md [hunk 20]
```diff
@@ -31,59 +31,69 @@ Get-EntraBetaGroupMember
...
+Connect-Entra -Scopes 'GroupMember.Read.All'
+Get-EntraBetaGroupMember -ObjectId 'eeeeeeee-4444-5555-6666-ffffffffffff'
 ```
-```output
+
+```Output
 Id                                   DeletedDateTime
 --                                   ---------------
-2ae2d97b-4bde-42aa-b7c0-7c91a4c91a77
+bbbbbbbb-7777-8888-9999-cccccccccccc
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupMember.md:49]:
```suggestion
eeeeeeee-4444-5555-6666-ffffffffffff
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaLifecyclePolicyGroup.md [hunk 21]
```diff
@@ -8,33 +18,47 @@ schema: 2.0.0
...
-PS C:\> Get-EntraBetaLifecyclePolicyGroup -Id cffd97bd-6b91-4c4e-b553-6918a320211c
+
+```Output
+Id                                   AlternateNotificationEmails GroupLifetimeInDays ManagedGroupTypes
+--                                   --------------------------- ------------------- -----------------
+bbbbbbbb-1111-2222-3333-cccccccccccc admingroup@contoso.com      200                 All
 ```
 
+This example demonstrates how to retrieve lifecycle policy object by Id in Microsoft Entra ID.  
 This command retrieves the lifecycle policy object to which a group belongs.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaLifecyclePolicyGroup.md:52]:
```suggestion
This example demonstrates how to retrieve lifecycle policy object for a group.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaUser.md [hunk 22]
```diff
@@ -8,78 +18,131 @@ schema: 2.0.0
...
+```Output
+DisplayName Id                                   Mail UserPrincipalName
+----------- --                                   ---- -----------------
+New User    cccccccc-2222-3333-4444-dddddddddddd      NewUser@contoso.com
+New User88  dddddddd-3333-4444-5555-eeeeeeeeeeee      demo99@contoso.com
 ```
 
-This command gets all the users whos title starts with sales.
-ie Sales Manager and Sales Assistant.
+This command gets all the users whose displayName starts with new.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaUser.md:136]:
```suggestion
This command gets all the users whose displayName starts with the word `New`.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUser.md [hunk 23]
```diff
@@ -19,136 +18,214 @@ schema: 2.0.0
...
 
-```output
-ObjectId                             DisplayName UserPrincipalName               UserType
---------                             ----------- -----------------               --------
-5e8b0f4d-2cd4-4e17-9467-b0f6a5c0c4d0 New user    NewUser@contoso.com             Member
+```powershell
+Connect-Entra -Scopes 'User.ReadWrite.All', 'Directory.ReadWrite.All'
+$PasswordProfile = New-Object -TypeName Microsoft.Open.AzureAD.Model.PasswordProfile
+$PasswordProfile.Password = '<Password>'
+New-EntraBetaUser -DisplayName 'New User' -PasswordProfile $PasswordProfile -UserPrincipalName 'NewUser@contoso.com' -AccountEnabled $true -MailNickName 'Newuser'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUser.md:77]:
```suggestion
$params = @{
    DisplayName = 'New User'
    PasswordProfile = $PasswordProfile
    UserPrincipalName = 'NewUser@contoso.com'
    AccountEnabled = $true
    MailNickName = 'Newuser'
}

New-EntraBetaUser @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUser.md [hunk 24]
```diff
@@ -19,136 +18,214 @@ schema: 2.0.0
...
 
-```output
-ObjectId                             DisplayName UserPrincipalName               UserType
---------                             ----------- -----------------               --------
-5e8b0f4d-2cd4-4e17-9467-b0f6a5c0c4d0 New user    NewUser@contoso.com             Member
+```powershell
+Connect-Entra -Scopes 'User.ReadWrite.All', 'Directory.ReadWrite.All'
+$PasswordProfile = New-Object -TypeName Microsoft.Open.AzureAD.Model.PasswordProfile
+$PasswordProfile.Password = '<Password>'
+New-EntraBetaUser -DisplayName 'New User' -PasswordProfile $PasswordProfile -UserPrincipalName 'NewUser@contoso.com' -AccountEnabled $true -MailNickName 'Newuser' -AgeGroup 'adult'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUser.md:107]:
```suggestion
$params = @{
    DisplayName = 'New User'
    PasswordProfile = $PasswordProfile
    UserPrincipalName = 'NewUser@contoso.com'
    AccountEnabled = $true
    MailNickName = 'Newuser'
    AgeGroup = 'adult'
}

New-EntraBetaUser @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUser.md [hunk 25]
```diff
@@ -19,136 +18,214 @@ schema: 2.0.0
...
 
-```output
-ObjectId                             DisplayName UserPrincipalName               UserType
---------                             ----------- -----------------               --------
-5e8b0f4d-2cd4-4e17-9467-b0f6a5c0c4d0 New user    NewUser@contoso.com             Member
+```powershell
+Connect-Entra -Scopes 'User.ReadWrite.All', 'Directory.ReadWrite.All'
+$PasswordProfile = New-Object -TypeName Microsoft.Open.AzureAD.Model.PasswordProfile
+$PasswordProfile.Password = '<Password>'
+New-EntraBetaUser -DisplayName 'New User' -PasswordProfile $PasswordProfile -UserPrincipalName 'NewUser@contoso.com' -AccountEnabled $true -MailNickName 'Newuser' -City 'New York'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUser.md:137]:
```suggestion
$params = @{
    DisplayName = 'New User'
    PasswordProfile = $PasswordProfile
    UserPrincipalName = 'NewUser@contoso.com'
    AccountEnabled = $true
    MailNickName = 'Newuser'
    City = 'New York'
}

New-EntraBetaUser @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUser.md [hunk 26]
```diff
@@ -19,136 +18,214 @@ schema: 2.0.0
...
 
-```output
-ObjectId                             DisplayName UserPrincipalName               UserType
---------                             ----------- -----------------               --------
-5e8b0f4d-2cd4-4e17-9467-b0f6a5c0c4d0 New user    NewUser@contoso.com             Member
+```powershell
+Connect-Entra -Scopes 'User.ReadWrite.All', 'Directory.ReadWrite.All'
+$PasswordProfile = New-Object -TypeName Microsoft.Open.AzureAD.Model.PasswordProfile
+$PasswordProfile.Password = '<Password>'
+New-EntraBetaUser -DisplayName 'New User' -PasswordProfile $PasswordProfile -UserPrincipalName 'NewUser@contoso.com' -AccountEnabled $true -MailNickName 'Newuser' -Department 'IT'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUser.md:167]:
```suggestion
$params = @{
    DisplayName = 'New User'
    PasswordProfile = $PasswordProfile
    UserPrincipalName = 'NewUser@contoso.com'
    AccountEnabled = $true
    MailNickName = 'Newuser'
    Department = 'IT'
}

New-EntraBetaUser @params
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUser.md [hunk 27]
```diff
@@ -268,13 +352,14 @@ Accept wildcard characters: False
 ```
 
 ### -ImmutableId
+
 This property is used to associate an on-premises user account to their Microsoft Entra ID user object.
 This property must be specified when creating a new user account in the Graph if you're using a federated domain for the user's userPrincipalName (UPN) property.
 
-Important: The $ and _ characters can't be used when specifying this property.
+Important: The $ and _ characters can't be used to when specifying this property.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUser.md:359]:
```suggestion
Important: The $ and _ characters can't be used when specifying this property.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupAppRoleAssignment.md (hunk 18 above)
Comment (snehalkotwal) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupAppRoleAssignment.md:50]: Changes done

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**